app/models/chargeback/report_options.rb

```diff
@@ -18,12 +18,24 @@ class Chargeback
     :ext_options,
     :include_metrics,      # enable charging allocated resources with C & U
     :method_for_allocated_metrics,
-    :group_by_tenant?
+    :group_by_tenant?,
+    :cumulative_rate_calculation,
   ) do
     def self.new_from_h(hash)
       new(*hash.values_at(*members))
     end
 
+    def skip_field_accumulation?(field, value)
+      return false unless cumulative_rate_calculation?
```

Comment:
The `unless` here is a little confusing. Would you mind flipping these to `return false if ...`?

Reply:
sure I also added comment to bring clarity in this method.
thanks!